packages/ilp-packet/src/errors.ts

```diff
@@ -1,105 +1,79 @@
 import { Writer } from 'oer-utils'
-import BaseError = require('extensible-error')
-
-export const codes = {
-  F00_BAD_REQUEST: 'F00',
-  F01_INVALID_PACKET: 'F01',
-  F02_UNREACHABLE: 'F02',
-  F03_INVALID_AMOUNT: 'F03',
-  F04_INSUFFICIENT_DESTINATION_AMOUNT: 'F04',
-  F05_WRONG_CONDITION: 'F05',
-  F06_UNEXPECTED_PAYMENT: 'F06',
-  F07_CANNOT_RECEIVE: 'F07',
-  F08_AMOUNT_TOO_LARGE: 'F08',
-  F99_APPLICATION_ERROR: 'F99',
-  T00_INTERNAL_ERROR: 'T00',
-  T01_PEER_UNREACHABLE: 'T01',
-  T02_PEER_BUSY: 'T02',
-  T03_CONNECTOR_BUSY: 'T03',
-  T04_INSUFFICIENT_LIQUIDITY: 'T04',
-  T05_RATE_LIMITED: 'T05',
-  T99_APPLICATION_ERROR: 'T99',
-  R00_TRANSFER_TIMED_OUT: 'R00',
-  R01_INSUFFICIENT_SOURCE_AMOUNT: 'R01',
-  R02_INSUFFICIENT_TIMEOUT: 'R02',
-  R99_APPLICATION_ERROR: 'R99',
-}
-
-export class BadRequestError extends BaseError {
-  public ilpErrorCode: string
-  constructor(message: string) {
-    super(message)
-    this.ilpErrorCode = codes.F00_BAD_REQUEST
-  }
+
+export enum IlpErrorCode {
+  F00_BAD_REQUEST = 'F00',
+  F01_INVALID_PACKET = 'F01',
+  F02_UNREACHABLE = 'F02',
+  F03_INVALID_AMOUNT = 'F03',
+  F04_INSUFFICIENT_DESTINATION_AMOUNT = 'F04',
+  F05_WRONG_CONDITION = 'F05',
+  F06_UNEXPECTED_PAYMENT = 'F06',
+  F07_CANNOT_RECEIVE = 'F07',
+  F08_AMOUNT_TOO_LARGE = 'F08',
+  F99_APPLICATION_ERROR = 'F99',
+  T00_INTERNAL_ERROR = 'T00',
+  T01_PEER_UNREACHABLE = 'T01',
+  T02_PEER_BUSY = 'T02',
+  T03_CONNECTOR_BUSY = 'T03',
+  T04_INSUFFICIENT_LIQUIDITY = 'T04',
+  T05_RATE_LIMITED = 'T05',
+  T99_APPLICATION_ERROR = 'T99',
+  R00_TRANSFER_TIMED_OUT = 'R00',
+  R01_INSUFFICIENT_SOURCE_AMOUNT = 'R01',
+  R02_INSUFFICIENT_TIMEOUT = 'R02',
+  R99_APPLICATION_ERROR = 'R99',
 }
 
-export class InvalidPacketError extends BaseError {
-  public ilpErrorCode: string
-  constructor(message: string) {
-    super(message)
-    this.ilpErrorCode = codes.F01_INVALID_PACKET
-  }
+export const codes = IlpErrorCode
```

Comment:
Can this be removed?

Reply:
Unfortunately not, it is part of the public API (via `export const Errors = errors` in `index.ts`) and it is being used at least in one place in `ilp-protocol-stream` and may be used in other packages and user code.